handler/create_scan/create_scan.go

```diff
@@ -71,13 +81,13 @@ func CreateScan(w http.ResponseWriter, r *http.Request) {
 
 	insertScanRes := tx.QueryRowxContext(r.Context(), `INSERT INTO public.scan_v1 (project_id, image_url, create_time) VALUES ($1, $2, NOW()) RETURNING scan_id`, reqBody.ProjectId, reqBody.ImageUrl)
 	var scanId uint64
-	if err := insertScanRes.Scan(&scanId); err != nil {
+	if err = insertScanRes.Scan(&scanId); err != nil {
 		w.WriteHeader(http.StatusInternalServerError)
 		log.Printf("failed to insert into scan table: %s", err.Error())
 		return
 	}
 
-	if err := tx.Commit(); err != nil {
+	if err = tx.Commit(); err != nil {
```

Comment:
this should be `:=`, weird it isn't

Reply:
That was actually what was causing teh bazel build to fail

![image](https://github.com/sunb26/xat/assets/56877739/df6d2b91-f4c0-49fe-8e17-732f0d8991c8)

Reply:
try fixing line 44? looks like it can be inlined in the condition too

Reply:
changed 44 and 58 to be inline but you get the same error becase err is assigned in line 71 still